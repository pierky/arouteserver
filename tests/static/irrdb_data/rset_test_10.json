{ "prefix_list": [
    { "prefix": "1.0.0.0\/16", "exact": false, "less-equal": 24 },
    { "prefix": "2.0.0.0\/16", "exact": false, "less-equal": 22 },
    { "prefix": "3.0.0.0\/16", "exact": true },
    { "prefix": "4.0.0.0\/8", "exact": false, "less-equal": 24, "greater-equal": 16 }
] }
