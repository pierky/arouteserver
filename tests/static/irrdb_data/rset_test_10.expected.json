[{
  "length": 16,
  "exact": false,
  "max_length": 32,
  "comment": null,
  "le": 24,
  "prefix": "1.0.0.0",
  "ge": null
 }, {
  "length": 16,
  "exact": false,
  "max_length": 32,
  "comment": null,
  "le": 22,
  "prefix": "2.0.0.0",
  "ge": null
 }, {
  "length": 16,
  "exact": true,
  "max_length": 32,
  "comment": null,
  "le": null,
  "prefix": "3.0.0.0",
  "ge": null
 }, {
  "length": 8,
  "exact": false,
  "max_length": 32,
  "comment": null,
  "le": 24,
  "prefix": "4.0.0.0",
  "ge": 16
 }]
